pkg/storage/repodb/repodbfactory/repodb_factory.go

```diff
@@ -0,0 +1,25 @@
+package repodbfactory
+
+import (
+	"zotregistry.io/zot/errors"
+	"zotregistry.io/zot/pkg/storage/repodb"
+)
+
+type RepoDBDriverFactory interface {
```

Comment:
Not a fan of the factory pattern when we only have a few things we can Create(), would just use switch/Create